akka-docs/src/main/paradox/stream/index.md

```diff
@@ -1,3 +1,6 @@
+---
+project.description: Streaming events with Akka Streams using stream graphs, stream flows failure handling and backpressure.
```

Comment:
Perhaps take the text from the akka.io/docs page (or something like it) instead
"An intuitive and safe way to do asynchronous, non-blocking backpressured stream processing."